getting-started/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt

```diff
@@ -35,7 +37,9 @@ object Chat {
         val localFocusManager = LocalFocusManager.current
 
         Column(
-            modifier = Modifier.fillMaxSize()
+            modifier = Modifier
+                .fillMaxSize()
+                .background(MaterialTheme.colors.surface)
```

Comment:
I'd rather use `background` color. 

A brief explaination:
`Surface colors map to components such as cards, sheets, and menus`
`Background color is found behind scrollable content`
source: [Material Theming](https://material.io/design/material-theming/implementing-your-theme.html#color)

Reply:
Change pushed ✔️